src/middlewared/middlewared/etc_files/ganesha/ganesha.conf

```diff
@@ -70,6 +70,13 @@ EXPORT_DEFAULTS
 
 % for share in shares:
     <%
+        if share['locked']:
```

Comment:
If we have locked datasets and unlocked datasets in the same share, we can still consider the share locked, but at the same time generate at least unlocked datasets?